Description: Disable coveralls while running tests
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2015-07-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -1,5 +1,5 @@
-require 'coveralls'
-Coveralls.wear!
+#require 'coveralls'
+#Coveralls.wear!
 
 RSpec.configure do |config|
   config.expect_with :rspec do |expectations|
